plugins/modules/azure_rm_postgresqlserver.py

```diff
@@ -192,7 +218,14 @@ def __init__(self):
             ),
             create_mode=dict(
                 type='str',
-                default='Default'
+                default='default',
+                choices=['default', 'geo_restore', 'point_in_time_restore']
+            ),
+            source_server_id=dict(
```

Comment:
@karisN What do these two parameters do “ source_server_id" and "restore_point_in_time"? Can you help to add a use case for these two parameters? Thank you very much!

Reply:
I updated the docs a bit more, hopefully that makes sense now.